platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLFCustoms.java

```diff
@@ -45,6 +47,21 @@
             // scale on Java 8 and Linux
             SPLIT_PANE_DIVIDER_SIZE_VERTICAL, UIScale.scale(4),
             SPLIT_PANE_DIVIDER_SIZE_HORIZONTAL, UIScale.scale(4),
+
+            // for org.openide.awt.CloseButtonFactory
```

Comment:
I remember from https://github.com/apache/netbeans/pull/859/files that the close-button drawing code had to be duplicated between and org.openide.awt.AquaVectorCloseButton and o.n.swing.tabcontrol.plaf.AquaVectorTabControlIcon. Are both of the corresponding locations for FlatLAF updated in this PR? Do the new UIManager keys prevent the need for this duplication?

Reply:
Yes, I thinks so. Also FlatLaf is a single module, but Aqua/Windows LAF support is scattered to many modules.
Anyway, there is only one close button in FlatLaf in `FlatTabControlIcon`.